python/llm/example/GPU/Deepspeed-AutoTP-FastAPI/run_llama2_7b_chat_hf_arc_2_card.sh

```diff
@@ -33,5 +33,4 @@ export TORCH_LLM_ALLREDUCE=0
 export WORLD_SIZE=2
```

Comment:
rename to start-deepspeed-autotp-ipex-llm-serving.sh?

Reply:
Resolved